nltools/stats.py

```diff
@@ -565,39 +557,61 @@ def two_sample_permutation(data1, data2, n_permute=5000,
     if return_perms:
         stats['perm_dist'] = all_p
     return stats
-
-
-def correlation_permutation(data1, data2, n_permute=5000, metric='spearman',
+        
+    
+def correlation_permutation(data1, data2, method='random', n_permute=5000, metric='spearman',
```

Comment:
I think `method = 'random'` is a bit confusing. Naively someone can assume you're using a random _method_ not randomly shuffling. Also random isn't informative as to what's happening. Maybe `method = 'shuffle'` or something like that?

Reply:
maybe shuffle?  they are all permutation methods. it's really elementwise, but I don't think that most people will get that.

Reply:
changed to permute, also caught another bug, where I forgot to add these methods to tests